File: gold-sample-app/src/main/java/com/gold/youtubeplayer/core/sampleapp/utils/VideoIdsProvider.java
package com.gold.youtubeplayer.core.sampleapp.utils;

import java.util.Random;

public class VideoIdsProvider {
    private static String[] videoIds = {"Orw8CZpzIDU", "51mE341m5X0", "upiS93DCR6o", "upiS93DCR6o"};
    private static String[] liveVideoIds = {"w4Ay4lHpR4o"};
    private static Random random = new Random();

    public static String getNextVideoId() {
        return videoIds[random.nextInt(videoIds.length)];
    }

    public static String getNextLiveVideoId() {
        return liveVideoIds[random.nextInt(liveVideoIds.length)];
    }
}

File: gold-sender-sample-app/src/main/java/com/gold/youtubeplayer/chromecast/sampleapp/utils/VideoIdsProvider.kt
package com.gold.youtubeplayer.chromecast.sampleapp.utils

import java.util.*

object VideoIdsProvider {
    private val videoIds = arrayOf("Orw8CZpzIDU", "51mE341m5X0", "upiS93DCR6o", "upiS93DCR6o")
    private val random = Random()

    fun getNextVideoId(): String {
        return videoIds[random.nextInt(videoIds.size)]
    }
}